cypress/e2e/testBoards.cy.js

```diff
@@ -0,0 +1,62 @@
+/// <reference types= "cypress" />
+
+
+describe("Boards", () => {
+	beforeEach(() => {
+        cy.fixture('example.json').then((value) => {
+            const email = value.email;
+            const pw = value.pw;
+            cy.login(email, pw);
+        });
+	});
+
+	describe("Create", function () {
+		it("Create a new board", () => {
+			//creating a board and assertion if it was created.
+			cy.fixture('example.json').then((value) =>{
+                const nameBoard = value.nameBoard;
+                cy.visit('https://trello.com/u/marcourquidi17/boards');
```

Comment:
The username can go in the example.json file as username and you can build this url with that.